src/Sulu/Bundle/MediaBundle/Tests/Functional/Entity/MediaDataProviderRepositoryTest.php

```diff
@@ -571,11 +569,16 @@ public function findByProvider()
         ];
     }
 
-    /**
-     * @dataProvider findByProvider
-     */
-    public function testFindByFilters($filters, $page, $pageSize, $limit, $expected, $tags = [], $options = []): void
-    {
+    #[\PHPUnit\Framework\Attributes\DataProvider('findByProvider')]
+    public function testFindByFilters(
+        array $filters,
+        ?int $page,
+        ?int $pageSize,
+        ?int $limit,
+        array $expected,
+        array $tags = [],
+        array $options = [],
+    ): void {
```

Comment:
test method changes should be done on 2.5 branch if possible and non breaking changes

I would keep it as oneliner